pyscf/lib/misc.py

```diff
@@ -1249,6 +1312,20 @@ def __enter__(self):
         temporary_env.__enter__(self)
         return self.c
 
+class h5filewrite_options(temporary_env):
+    '''Within the context of this macro, extra keyword arguments are
+    passed to h5py.File() whenever an HDF5 file is opened for writing.
+
+    Examples:
+
+    >>> with h5filewrite_options(alignment_interval=4096, alignment_threshold=4096):
+    ...     f = h5py.File('mydata.h5', 'w')
```

Comment:
Isn't it `f = lib.H5FileWrap('mydata.h5', 'w')` ?

Reply:
You're right. My bad!

Reply:
Fixed now.